clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchResponse.java

```diff
@@ -113,8 +120,14 @@ public int hashCode() {
         }
     }
 
+    public OffsetFetchResponse(OffsetFetchResponseData data) {
+        super(ApiKeys.OFFSET_FETCH);
+        this.data = data;
+        this.error = null;
+    }
+
     /**
-     * Constructor for all versions without throttle time.
+     * Constructor without throttle time for version 0 to version 7.
```

Comment:
It is not actually for version 0 to 7, it is for the versions without throttle time.